src/core/engines/Cesium/Feature/Tileset/hooks.ts

```diff
@@ -119,6 +124,10 @@ const TILESET_APPEARANCE_FIELDS: (keyof Cesium3DTilesAppearance)[] = [
 const convertStyle = (val: any, convert: StyleProperty["convert"]) => {
   if (convert === "color") {
     return toColor(val);
+  } else if (convert === "colorFunctionString") {
+    return `color("${val}")`;
+  } else if (convert === "booleanString") {
+    return typeof val === "boolean" ? (val ? "true" : "false") : val;
```

Comment:
Why does `show` need to be string?

Reply:
I have updated this part removed the convert for boolean show.
I use this convert to avoid it been igored in `MODEL_STYLE_PROPERTIES.reduce`, there was `if(!val) return res;` and now i update it as `if (val === undefined) return res;` Is this okey? I'm not sure maybe i could miss something here.